framework/src/org/checkerframework/common/aliasing/qual/MaybeAliased.java

```diff
@@ -24,6 +24,6 @@
 @DefaultQualifierInHierarchy
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.TYPE_PARAMETER, ElementType.TYPE_USE })
-@ImplicitFor(trees = { Tree.Kind.NULL_LITERAL }, typeNames = { java.lang.Void.class })
+@ImplicitFor(literals = { LiteralKind.NULL }, typeNames = { java.lang.Void.class })
```

Comment:
Many times LiteralKind.NULL appears together with typeName j.l.Void, but not always.
Is that an inconsistency that we should fix? (This is unrelated to your changes, but would be a good change.)